src/express-http-server.js

```diff
@@ -17,6 +17,12 @@ class ExpressHTTPServer {
     this.gzip = options.gzip || false;
     this.beforeMiddleware = options.beforeMiddleware || noop;
     this.afterMiddleware = options.afterMiddleware || noop;
+    this.forbiddenAssets = [
```

Comment:
I think it would be useful to have `options.forbiddenAssets` here as well to allow customization.